components/tidb_query/src/codec/row/v2/encoder.rs

```diff
@@ -49,6 +49,7 @@ pub struct Column {
     ft: FieldType,
 }
 
+// TODO: refactor
```

Comment:
Please explain it a little more clear about how should we refactor it.

Reply:
sorry, forgot to delete this line 😄